Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo

```diff
@@ -0,0 +1,85 @@
+within Buildings.ChillerWSE.BaseClasses.Examples.BaseClasses;
+partial model PartialPlantWithControl
+  "Partial model that tests source plant with temperature control"
+  import Buildings;
```

Comment:
@YangyangFu You may have deleted the `Buildings.ChillerWSE.BaseClasses.Examples` folder using Dymola and it did not show up when opening the Building library in Dymola. But the file folder `modelica-buildings/Buildings/ChillerWSE/BaseClasses/Examples/` is still in your branch. You should manually delete the folder.